core/db/migrate/20180405181459_add_default_state_to_inventory_unit.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultStateToInventoryUnit < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:spree_inventory_units, :state, Spree::InventoryUnit::ON_HAND)
```

Comment:
We should use the value of the constant here so we do not rely on the code may change in the future.